src/requests/init.jl

```diff
@@ -251,8 +251,10 @@ function initialized_notification(params::InitializedParams, server::LanguageSer
                 file_full_path = joinpath(server.env_path, file)
                 uri = filepath2uri(file_full_path)
                 if isfile(file_full_path)
+                    # TODO properly normalize drive letters in paths
+                    file_full_path_lowerfirst = lowercasefirst(file_full_path)
```

Comment:
This won't work for UNC paths, i.e. paths that start with `\\SERVER\share\...`, and we also shouldn't do this on Linux. Maybe something like
```
@static if Sys.iswindows()
  if length(file_full_path) > 1 && isletter(file_full_path[1]) && file_full_path[2] == ':'
    file_full_path = lowercasefirst(file_full_path)
  end
end
```

Reply:
Updated in https://github.com/julia-vscode/LanguageServer.jl/pull/1313/commits/ba36759555a2f8ff88c8016b31e49e8d8e5794b6